### ISSUE TITLE
Unable to login with WindowsAzureActiveDirectory

### ISSUE THREAD

**Issue by mirogrg1** [MicrosoftDocs/xamarin-docs#1237]
I have used pretty much the same code as detailed in this page, using Microsoft.Identity.Client v 1.1.2. I am able to get an access token but when I try to use that with MobileServiceClient.LoginAsync for WindowsAzureActiverDirectory provider I get an exception like this: Microsoft.WindowsAzure.MobileServices.MobileServiceInvalidOperationException: The page cannot be displayed because an internal server error has occurred. 

What is not configured correctly here? This is supposed to be using the client flow and I used the sample app above as an example. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 16864cab-cbfa-98f3-c1da-7ed88cfe2579
* Version Independent ID: c1731783-e6da-af46-3fce-5e4bcc50dd61
* Content: [Integrating Azure Active Directory B2C with Azure Mobile Apps - Xamarin](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/data-cloud/authentication/azure-ad-b2c-mobile-app)
* Content Source: [docs/xamarin-forms/data-cloud/authentication/azure-ad-b2c-mobile-app.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/xamarin-forms/data-cloud/authentication/azure-ad-b2c-mobile-app.md)
* Product: **xamarin**
* GitHub Login: @davidbritch
* Microsoft Alias: **dabritch**

**Comment by davidbritch** [comment]
Hi @mirogrg1 

As you're getting an access token from ADB2C, the main thing to check is that your Azure Mobile Apps instance can communicate with your ADB2C tenant. For more info, see https://docs.microsoft.com/xamarin/xamarin-forms/data-cloud/authentication/azure-ad-b2c-mobile-app#azure-mobile-app-configuration

**Comment by mirogrg1** [comment]
That did help to narrow it down. I had used the wrong issuer URL but I guess I was very confused by what that URL really is. I finally realized the correct one was at the top of the "Run user flow" window in the ADB2C tenant. The whole time I was using "Run user flow endpoint" URL because the keyword for me was "endpoint" so I thought it was the one I needed. The URL at the top is the correct one but not obvious what exactly it means.
